Commands/security/api-collection/readme.md

```diff
@@ -0,0 +1,17 @@
+# [Group] _security api-collection_
+
+Manage Azure API Management API connections to Microsoft Defender for APIs.
+
+## Commands
+
+- [create](/Commands/security/api-collection/_create.md)
```

Comment:
Can we use a different command than "create", like "onboard-api-mgmt-api"? "create" is too generic and will not account for future scenarios where we may want to onboard apiCollections from other resources like App Services or K8s.